### PR TITLE
Rename typed-frames to iskra

### PR DESCRIPTION
**Pull request by romanowski**

### CODE REVIEW

**Comment by tgodzik** [comment]
Isn't it too polish? :sweat_smile:

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# TypedFrames
+# Iskra
 
 TypedFrames is a Scala 3 wrapper around Apache Spark API which allows writing typesafe and boilerplate-free but still efficient Spark code.
```
Comment (prolativ) [inline README.md:3]:
```suggestion
Iskra is a Scala 3 wrapper around Apache Spark API which allows writing typesafe and boilerplate-free but still efficient Spark code.
```

**Review by prolativ — CHANGES_REQUESTED** [review]

**Comment by prolativ** [comment]
Two other things to be done:
1) Update `publish-conf.scala`
2) Rename packages
The second point could be done in a separate PR as well

**Comment by prolativ** [comment]
> 



> Isn't it too polish? 😅

Wasn't the previous name too English? 😉 

 
--

**Comment by tgodzik** [comment]
Maybe let's polish it up a bit more. I think `trzcina` is a great name :sweat_smile:

**Review by prolativ — APPROVED** [review]

**Pull request merged**